cypress/tests/ImageElement.spec.ts

```diff
@@ -120,6 +120,21 @@ describe("ImageElement", () => {
         getElementField("useSrc").find("input").click();
         assertDocHtml(getSerialisedHtml({ useSrcValue: "false" }));
       });
+
+      it(`should track the field offset, and update correctly after fields above have changed`, () => {
+        addElement();
+
+        getElementField("useSrc").find("input").click();
+        typeIntoElementField("altText", "Example text");
+        getElementField("useSrc").find("input").click();
```

Comment:
I think this test could be improved, given the `useSrcValue` represents a boolean value it's possible to imagine this test passing if both the checkbox clicks fail.  We could possibly improve by using a dropdown field instead or asserting between checkbox clicks.

Reply:
Great point. Could give this a tweak once #41 is merged, perhaps!